adafruit_matrixportal/matrix.py

```diff
@@ -70,23 +66,33 @@ def __init__(self, *, width=64, height=32, bit_depth=2, alt_addr_pins=None):
                 board.MTX_G2,
                 board.MTX_B2,
             ]
-        else:
-            rgb_pins = [board.D2, board.D3, board.D4, board.D5, board.D6, board.D7]
-
-        if hasattr(board, "MTX_CLK"):
             clock_pin = board.MTX_CLK
-        else:
-            clock_pin = board.A4
-
-        if hasattr(board, "MTX_CLK"):
             latch_pin = board.MTX_LAT
-        else:
-            latch_pin = board.D10
-
-        if hasattr(board, "MTX_OE"):
             oe_pin = board.MTX_OE
-        else:
+        elif hasattr(board, "D7"):
+            # Metro/Grand Central Style Board
```

Comment:
i think maybe better to look in os.uname?

Reply:
Cool, I looked for something like that but didn't see it the first time. Thanks.